util/checkers/number.go

```diff
@@ -52,3 +52,19 @@ func DecInZeroOne(a sdk.Dec, name string, oneInclusive bool) error {
 	}
 	return nil
 }
+
+// DecNotNegative checks if a is defined and a >= 0
+func DecNotNegative(a sdk.Dec, paramName string, errs []error) []error {
```

Comment:
For these functions, they should probably just return `error` and any `append(errs, err)` can be kept outside of them. This will keep them more standard in case we want to use them anywhere else